### ISSUE TITLE
Cannot label Frames

### ISSUE THREAD

**Issue by Astercan** [DeepLabCut/DeepLabCut#2705]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Hi, nothing happens after labeling my frames. No folder is created, hence I cannot create the training dataset. 
When the GUI starts, I get this message on Terminal:

> qt.qpa.drawing: Layer-backing is always enabled.  QT_MAC_WANTS_LAYER/_q_mac_wantsLayer has no effect.

This message is repeated every time I click on something. 
When I click on "label frames:, I get this message: 

> WARNING: Layer-backing is always enabled.  QT_MAC_WANTS_LAYER/_q_mac_wantsLayer has no effect.
> WARNING:vispy:Layer-backing is always enabled.  QT_MAC_WANTS_LAYER/_q_mac_wantsLayer has no effect.

How could I fix this? The odd thing is that sometimes it does work, so this seems to happen purely by chance. 

Thank you

### Operating System

Mac Sonoma M1 

### DeepLabCut version

DLC 2.3.9

### DeepLabCut mode

single animal

### Device type

CPU

### Steps To Reproduce

1. Anaconda navigator / Deeplabcut M1 / Open terminal
2. pythonw -m deeplabcut
3. First warning appears on terminal : qt.qpa.drawing: Layer-backing is always enabled.  QT_MAC_WANTS_LAYER/_q_mac_wantsLayer has no effect.
4. Create New Project / Edit config.yaml (2 bodyparts)
5. Extract Frames. These are created without issues, though the message "qt.qpa.drawing: Layer-backing is always enabled.  QT_MAC_WANTS_LAYER/_q_mac_wantsLayer has no effect." appears on every click
6. Label Frames. Now the message "WARNING: Layer-backing is always enabled. QT_MAC_WANTS_LAYER/_q_mac_wantsLayer has no effect." appears
7. No folder is created with labeled frames. Unable to do the training

### Relevant log output

```shell
qt.qpa.drawing: Layer-backing is always enabled.  QT_MAC_WANTS_LAYER/_q_mac_wantsLayer has no effect.
WARNING: Layer-backing is always enabled.  QT_MAC_WANTS_LAYER/_q_mac_wantsLayer has no effect.
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by MMathisLab** [comment]
are you sure your in the conda env? as QT is not a requirement :D

**Comment by Astercan** [comment]
Thank you for your reply :)
I am not versed on these environments, so I do not even know what QT is or why it is there, but I have the same issue (no labeled frames are created) on either a Mac Mini or a Windows desktop.
On both computers I start DLC from **Anaconda Navigator**. I go to **Environment / Deeplabcut** (Environment / Deeplabcut_M1 on Mac). 
On **Mac** I go to "Open Terminal", then write "`pythonw -m deeplabcut`"
On **Windows** I go to "Open with iPython", then type "`import deeplabcut`"

On **Windows** I get the following message:

> 2024-08-15 10:32:28.025583: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'cudart64_110.dll'; dlerror: cudart64_110.dll not found
> 2024-08-15 10:32:28.026277: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.

I type "`deeplabcut.launch_dlc()`" and get the following message:

> The syntax of the command is incorrect.
> 3382it [00:49, 67.75it/s]

But the interface opens regardless.  

On **Mac** I get the messages I posted earlier. 

Is there anything I should be doing differently? 
As I said before, I have no clue what these error messages mean, or what QT is (or why it is there).

Thank you!

**Comment by Astercan** [comment]
@MMathisLab
            
Hi, I noticed that this question was closed, but I'm still encountering the same issue. I cannot create labeled frames and could really use some guidance on how to resolve it. I've also encountered this error on a Windows computer, which I mentioned in my previous post. Any insights would be greatly appreciated.

Thank you!

**Comment by MMathisLab** [comment]
HI @Astercan I don't have a solution aside from downgrading then to match the DLC version you want to use; i.e. downgrade to something ~ 2.1.5: https://pypi.org/project/napari-deeplabcut/#history. I closed it as I have no other suggestions and we don't patch old versions (it's just way too much work for a small lab). 

The current version works on windows & the latest pytorch version is higher performance, so I would consider switching or using the older GUI for labeling that you must have used beforehand

**Comment by Astercan** [comment]
@MMathisLab
Thank you, I was not aware that was the case. I did not even know there was a newer version; will try that one.

Thanks again
